charts/opensearch-dashboards/CHANGELOG.md

```diff
@@ -13,6 +13,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 ### Security
 ---
+## [1.3.1]
+### Added
+- Add pod autoscaling capability.
+- Added autoscaling configuration parameters in readme.
```

Comment:
Nit: Can we maintain a common tense here for better readability?

Reply:
Fixed in latest commit.